File: play-2.5.x/K-Modules/kmodules/K-Base/app/K/Controllers/ApiDoc/TemplateModel/ApiGroup.java
package K.Controllers.ApiDoc.TemplateModel;

import K.Common.Helper;
import K.Controllers.ApiDoc.Reply.ApiInfo;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by kk on 15/1/12.
 */
public class ApiGroup {
    public String groupName;
    public List<ApiInfo> apiInfoList;

    public ApiGroup(String group_name) {
        groupName = group_name;
        apiInfoList = new ArrayList<>();
    }

    public String ToMarkdownStr(String str) {
        return Helper.EscapeMarkdown(str);
    }


}

File: play25kotlin/subProjects/ToolBox/app/k/common/apidoc/ApiGroup.kt
package k.common.apidoc


import k.common.Helper

/**
 * Created by kk on 15/1/12.
 */
class ApiGroup(val groupName: String) {

    var apiInfoList: MutableList<ApiInfo>

    init {
        apiInfoList = mutableListOf<ApiInfo>()
    }

    fun ToMarkdownStr(str: String): String {
        return Helper.EscapeMarkdown(str)
    }


}
